tests/testcases/030_check-network.yml

```diff
@@ -54,7 +54,28 @@
     delay: 5
 
   - name: Run 2 busybox pods in test ns
```

Comment:
If that is succeeded, all `busybox` words are necessary to be replaced in this file.